.github/workflows/test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         run: /entrypoint.sh --nopatch t/unit_fixture 2>/dev/null
 
       - name: Run unit_mech tests
-        run: /entrypoint.sh --nopatch t/unit_mech 2>/dev/null
+        run: /entrypoint.sh --nopatch t/unit_mech
```

Comment:
@MFlores2021 @lukasmueller what are your thoughts on removing the `2>/dev/null` from the test runs?  This would allow the output from the tests to be visible in GH Actions, which can be helpful for debugging failed tests.